# This is a patch for hyperion to update it to hyperion.patched
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -Naud 'hyperion/CMake/Herc60_CreateTargets.cmake' 'hyperion.patched/CMake/Herc60_CreateTargets.cmake'
Index: ./CMake/Herc60_CreateTargets.cmake
--- ./CMake/Herc60_CreateTargets.cmake	Tue Jul 12 15:20:04 2022
+++ ./CMake/Herc60_CreateTargets.cmake	Tue Jul 12 15:20:39 2022
@@ -409,7 +409,7 @@
 
 externalproject_add( html
         SOURCE_DIR        ${PROJECT_BINARY_DIR}/html
-        GIT_REPOSITORY    "git://github.com/hercules-390/html"
+        GIT_REPOSITORY    "https://github.com/hercules-390/html"
         GIT_TAG           "gh-pages"
         CONFIGURE_COMMAND ""        # No Configure
         BUILD_COMMAND ""            # No build
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Tue Jul 12 15:24:46 2022
# Generated by        : makepatch 2.03
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p 'CMake/Herc60_CreateTargets.cmake' 18214 1657660839 0100664
#### End of ApplyPatch data ####

#### End of Patch kit [created: Tue Jul 12 15:24:46 2022] ####
#### Patch checksum: 47 1919 52478 ####
#### Checksum: 65 2604 44232 ####
